shared/js/smart-screen/bookmark_manager.js

```diff
@@ -0,0 +1,91 @@
+/* global CardStore, evt */
+
+(function(exports) {
+  'use strict';
+
+  var bookmarkManager = evt({
```

Comment:
Please capitalize the first letter (`BookmarkManager`) so that it can be integrated with `BaseModule` in the near future.